lib/ember-test-helpers/isolated-container.js

```diff
@@ -66,6 +66,18 @@ export default function isolatedContainer(fullNames) {
   container.register('view:select', Ember.Select);
   container.register('route:basic', Ember.Route, { instantiate: false });
 
+  var global = typeof global === 'object' && global || window;
```

Comment:
Because `var global` is hoisted, I don't think you can make this `typeof global` check it should work if you choose a different local variable name though.  Also, `self` should be used instead of `window` (as it works in WebWorkers and normal browser scenarios, but `window` does not work in WebWorkers).

I'm thinking of something like:

```javascript
var globalContext = typeof global === 'object' && global || self;
```